python_modules/dagster/dagster/_core/definitions/declarative_scheduling/scheduling_condition.py

```diff
@@ -40,6 +40,20 @@
     from .scheduling_context import SchedulingContext
 
 
+# static factory
+class Scheduling:
```

Comment:
I'm of two minds on this pattern -- there are two nice benefits of making this a separate class (as I see it):

1. Less verbose for common cases
2. Makes it clear which conditions are "basic building blocks" and which are more complex sequences (so you don't get surprised when you go to the UI and see a whole nested structure when all you did was type "SchedulingCondition.on_cron()"

But theoretically this adds another concept that needs to be explained to the user (with very similar names)

I land on the side of including this for now -- we can always revise later but it does net out to feeling a bit smoother.